docs-src/content/using-cubic-sdk/client-configs.md

```diff
@@ -0,0 +1,108 @@
+| enable_word_time_offsets | No | `false` | Toggles the calculation of word-level timestamps.  Cubic models must support word level timestamps for this field to be populated. |
+| enable_word_confidence | No | `false` | Toggles the calculation of word-level confidence scores. Cubic models must support word level confidence for this field to be populated.|
+| enable_raw_transcript | No | `false` | If `true`, the raw transcript will be included in the results. |
+| enable_confusion_network | No | `false` | Toggles the creation of a confusion network, consisting of multiple alternatives transcriptions.  Cubic models must support confusion networks for this field to be populated. |
+| audio_channels | No | `[0]` (mono) | Specifies which channels of a multi-channel audio file to be transcribed, each as their own individual audio stream. |
+
+## Use cases
+
+The most basic use case is getting a formatted transcript.  This would simply need a config such as:
+
+```json
+{
+    "model_id": "1",
+    "audio_encoding": "raw"
+}
+```
+
+and your resulting transcript would be found at `results.alternatives[*].transcript`.
+
+> Note: Prior to `cubicsvr` v2.9.0 and SDK-Cubic v1.3.0, the field `results.alternatives[*].transcript` was populated with either the raw or the formatted transcription depending on the `enable_raw_transcript` config.  After these changes, raw transcripts have been pushed to a new field `results.alternatives[*].raw_transcript`, and only populated when `enable_raw_transcript` is set to true.
+
+------
+
+Sometimes, only the Raw transcript is desired.  In this case, there are two options.  
+
+1. Disable server-side formatting and use the above config.  Retrieve the raw transcript from the `results.alternatives[*].transcript` field.
+2. Specify the `enable_raw_transcript = true` flag, and access the field `results.alternatives[*].raw_transcript`.
+
+> Note: Prior to `cubicsvr` v2.9.0 and SDK-Cubic v1.3.0, the field `results.alternatives[*].transcript` was populated with either the raw or the formatted transcription depending on the `enable_raw_transcript` config.  After these changes, raw transcripts have been pushed to a new field `results.alternatives[*].raw_transcript`, and only populated when `enable_raw_transcript` is set to true.
+
+------
+
+Another use case would be getting both the formatted and raw transcript.  This can be done using this config.
+
+```json
+{
+    "model_id": "1",
+    "audio_encoding": "raw",
+    "enable_raw_transcript": "true"
+}
+```
+
+The formatted transcript would be found at `results.alternatives[*].transcript`, and the raw transcript would be found at `results.alternatives[*].raw_transcript`.
+
+------
+
+If you needed to dive deeper in word level timestamps, say for subtitles, then you can use this config to enable those word-level timestamps.  Please note the inclusion of `enable_raw_transcript`.
```

Comment:
```suggestion
If you need to know the timestamp for each word, to align subtitles with a video, for example, then you can use this config to enable those word-level timestamps.  Please note the inclusion of `enable_raw_transcript`; the word-level information corresponds to the raw transcript, since the formatter may combine multiple words into one symbol in the formatted transcript (e.g. "twenty one" to "21").
```
For Exxon, we implemented providing word-level info for the formatted output.  Is that only implemented for them or should that be an option in cubicsvr directly?  I don't know if there's a pressing need for it, I'm just wondering.

Reply:
I don't see why we wouldn't support it for everyone, but that's just me.  I'm not sure how we want to handle the format of things though, now that we support raw and formatted side by side.